Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM alpine:3.11
+
+RUN apk add --no-cache coreutils curl jq gfortran libc-dev cmake make python3 git
```

Comment:
I've added `jq` and `curl` as dependencies. `curl` is used below to download the testing library and `jq` is used later to write a nicely formatted `results.json` file.